inc/commondbtm.class.php

```diff
@@ -347,7 +347,7 @@ public function getFromDBByCrit(array $crit) {
       } else if (count($iter) > 1) {
          Toolbox::logWarning(
             sprintf(
-               'getFromDBByCrit expects to get one result, %1$s found!',
+               'getFromDBByCrit expects to get one result, %1$s found! '.$iter->getSql(),
                count($iter)
             )
```

Comment:
Maybe using `trigger_error()` could help to have a better trace.
```php
trigger_error(
   sprintf(
      'getFromDBByCrit expects to get one result, %1$s found in query "%2$s".',
      count($iter),
      $iter->getSql()
   ),
   E_USER_WARNING
);
```

Reply:
This will change the behaviour; and may break existing cases

Reply:
It will not be blocking, so it will not change the behaviour.

Reply:
OK, as we've discussed